### ISSUE TITLE
OSKAR Noise Implementation fails in segfault on specific parameters.

### ISSUE THREAD

**Issue by cvoegele** [i4Ds/Karabo-Pipeline#135]
Running the test /test/test_system_noise.py results in a Killed Signal.
While trying to find out what's happening, I noticed that it most likely has to do with the configuration of the Noise in the OSKAR Input.

Running OSKAR standalone with the following configuration works.

```
app=oskar_sim_interferometer
version=2.7.0

[simulator]
double_precision=false

[sky]
oskar_sky_model\file=sky.osm

[observation]
num_channels=3
start_frequency_hz=100e6
frequency_inc_hz=20e6
phase_centre_ra_deg=20
phase_centre_dec_deg=-30
num_time_steps=24
start_time_utc=01-01-2000 12:00:00.000
length=12:00:00.000

[telescope]
input_directory=telescope.tm
aperture_array\element_pattern\enable_numerical=false

[interferometer]
oskar_vis_filename=example.vis
ms_filename=test.ms
channel_bandwidth_hz=1e6
time_average_sec=1
noise/enable=False
noise/seed=1
noise/freq/inc=1.e8
noise/freq/number=24
noise/freq=Telescope
noise/rms=Telescope
noise/freq/start=1.e9
noise/rms/start=5000
noise/rms/end=10000
```

However if you change the noise parameters to the defaults and then set enabled to False it segfaults. File like this.

```
app=oskar_sim_interferometer
version=2.7.0

[simulator]
double_precision=false

[sky]
oskar_sky_model\file=sky.osm

[observation]
num_channels=3
start_frequency_hz=100e6
frequency_inc_hz=20e6
phase_centre_ra_deg=20
phase_centre_dec_deg=-30
num_time_steps=24
start_time_utc=01-01-2000 12:00:00.000
length=12:00:00.000

[telescope]
input_directory=telescope.tm
aperture_array\element_pattern\enable_numerical=false

[interferometer]
oskar_vis_filename=example.vis
ms_filename=test.ms
channel_bandwidth_hz=1e6
time_average_sec=1
noise/enable=True
noise/seed=1
noise/freq/inc=0
noise/freq/number=0
noise/freq=Range
noise/rms=Range
noise/freq/start=0
noise/rms/start=0
noise/rms/end=0
```

Its hard to track. Maybe pinging the guys at OSKAR would be a good idea.

**Comment by kenfus** [comment]
This seems to be correct. The test `test_create_detection_from_ms` fails currently but with standard parameters provided by @rohitcbscient it works. I will update the standard parameters in the function `InterferometerSimulation`.

**Comment by deiruch** [comment]
Duplicate of #117

**Comment by kenfus** [comment]
I will add the fix from #117 here, so that the history is more clean.
